README.md

```diff
@@ -71,7 +71,10 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+Maybe model method.
+ I feel like it is model method because of separation of concerns. You want to have your logic in the model and not the controller so that your controller only controls where things go.
+You could create a custom helper but an inline link_to might work, however, inline code, if I remember correctly is bad.
+So than finally back to model method, you could create a method that checks for the avg_rating number and translates that to which css rule to use.
 ```
```

Comment:
The correct answer here is a **helper method.** Helper methods are useful in rendering something in the view based on certain conditions. Model methods are most often used when doing something to data as it is pulled out the database (i.e., defined in a model, then used in a controller or model).